### ISSUE TITLE
All profiles should be reviewed

### ISSUE THREAD

**Issue by stuartpb** [opws/domainprofiles#19]
I want to mandate the presence of the "Reviewed" field for all profiles (so at least they need to be reviewed before inclusion). This will entail reviewing all existing profiles:

*(checklist removed)*

**Comment by stuartpb** [comment]
Structural aspects that need to be checked / fixed when reviewing profiles:

- #4 (only a handful of profiles)
- #12 (any non-identifier strings must be preceded by `- en:`)

**Comment by stuartpb** [comment]
Okay, quick back-of-the-envelope calculation says I can knock all of these out by the end of the month if I accomplish five reviews a day.

That seems... *maybe* manageable, but, of course, the big thing is, what do I want a review to *entail*? The reason I've made so little progress, really, is because each review I've done has involved so much *other* stuff, like, in a lot of cases, figuring out what *happened* to a site, or how something I wrote *before* relates to what I'm doing *now*, and stuff like that.

So... ech.

**Comment by stuartpb** [comment]
FWIW, what I'm finding right now is not so much that I'm getting blocked on the *reviews themselves*, but that I'm getting swamped by all the *refactoring* I'm doing between reviews, including all the meta-natter that I'm putting into issues. Like, I'm four profiles behind schedule right now, and that's because, for the last few days, I've been up several hours past midnight spittin' proposals and cross-references and stuff like that trying to keep track of what I'm *not* recording. (I think *just coming up with the `WILDCARD` solution* took me up to 11 PM on Tuesday night.)

That said, previously I've let that *stop* me from working on this, and, well, this month it's my Project of the Month, so I'm going to stick with it, and I'll see what keeps popping up in the long term, and, well, we'll see.

**Comment by stuartpb** [comment]
Okay, so, I've made pretty good progress on this so far (there are 25 remaining to be profiled, and, wow, I'm pretty sure that number was over 100 before I started), doing these reviews has given me a lot of insight into how the schema should be drafted, and I've hit on a *really solid* workflow for putting these reviews together in the future.

However, as much as I *want* to review the 25 remaining unreviewed profiles before the 28th, I'd rather spend the rest of this month closing out a bunch of the *other* remaining issues on this project, including splitting it up into separate repositories, and writing a proper schema with validation.

As such, I've decided I'm going to chump out and add reviewed dates to any remaining profiles that were added individually based on the timestamp of their first commit (or the last commit that meaningfully updated their information), and then only immediately review any that are still unreviewed after *that*. (A new issue can be opened up once this is done to review all the profiles that got retroactive review dates based on a "review everything before April 15 2015" filter, which can then be done post-February.)

**Comment by stuartpb** [comment]
Okay, I'm merging this in now, as #280. (I'm holding back a couple profiles that I intend to either remove or move with review, as those will be separate PRs.) Every site that was added with its own commit (including everything after this project was first established) uses the date of that commit as its timestamp; every site that was added in the original blot.pw list in a *monolithic* commit adding *multiple* sites uses the date of the *prior* commit (as we can't know for certain that it was any *later*).

For reference, this is the breakdown of how the backfill dates were established:

## Backfilled from domainprofiles commit
- [x] [flattr.com](https://github.com/opws/domainprofiles/commit/0e555d73d46c31e7d0d53a0fd6f456bdffa63edb)
- [x] [diigo.com](https://github.com/opws/domainprofiles/commit/3efb5d3a4bf1623c9098cd8bff23b8e7e8669fcc)
- [x] [disneyworld.disney.go.com](https://github.com/opws/domainprofiles/commit/adad7d676768fcba9fa0f1be2420359498333c36)
- [x] [trello.com](https://github.com/opws/domainprofiles/commit/8d8f48f810afa9a4f39fb87e9ef4356301bfe955)
- [x] [iwantmyname.com](https://github.com/opws/domainprofiles/commit/ab31343979aa138ae1eca1d0f0eabb712a4c341e)
- [x] [topcoder.com](https://github.com/opws/domainprofiles/commit/3f4fcb349a7441500e95c545898e39b7b28b20a7)
- [x] [zipcar.com](https://github.com/opws/domainprofiles/commit/1b3098dcce0e65b565c25730d86720c8e81a4c09)
- [x] [gravatar.com](https://github.com/opws/domainprofiles/commit/c4187f42bf09fffe81340ffdc7a3f853e66e940c)
- [x] [wordpress.com](https://github.com/opws/domainprofiles/commit/126b01d393844028ef081389b086538c2ec88c38)

## Backfilled from atomic blot.pw commit
- [x] [freesound.org](https://github.com/blotpass/blot.pw/commit/df965920277e8b649765ecc1494a113c75f640a8)
- [x] [koding.com](https://github.com/blotpass/blot.pw/commit/1cf76eebbc15d5769a7632d428bad3580bf4fcdc)
- [x] [wikimedia.org, wikipedia.org, and wikivoyage.org](https://github.com/blotpass/blot.pw/commit/f3a8b86ac9a82faab68f2aa86cce72bcd26df54a)

## Backfilled to 2013-10-22T06:29:22Z, [committed at 2013-10-25T18:19:02Z](https://github.com/blotpass/blot.pw/commit/e0b74581d87338728ea9dbb485674760717cf31c)
- [x] eventbrite.com
- [x] guru.com

## Backfilled to 2013-06-16T01:33:52Z, [committed at 2013-09-09T06:25:54Z](https://github.com/blotpass/blot.pw/commit/cd5a534ee2ab971d50625902ab16ec1d9393e4cd)
- [x] laptopscreen.com
- [x] digitalocean.com

## Backfilled to 2013-05-30T17:41:28Z, [committed at 2013-06-16T01:33:52Z](https://github.com/blotpass/blot.pw/commit/2ca8dcef435d5e2d796b615e56ea0affb42a255b)
- [x] html5-ninja.com
- [x] twilio.com
- [x] projecteuler.net
- [x] wibit.net

## Backfilled to 2013-05-06T11:07:47Z, [committed at 2013-05-22T13:09:06Z](https://github.com/blotpass/blot.pw/commit/f581afc5bf3a3b431750b4f421e1a0c0bd5b1eb7)
- [x] yahoo.com
- [x] linode.com

**Comment by stuartpb** [comment]
Okay, now that #280, #281, #282, #284, and #285 have closed, all profiles have a `reviewed` field (as far as I know), so this issue, in its current form, can finally be closed (with [the potential for a similar issue to be opened later](https://github.com/opws/domainprofiles/issues/19#issuecomment-282520004)).
